src/QsCompiler/Tests.Compiler/ClassicalControlTests.fs

```diff
@@ -9,7 +9,7 @@ open Microsoft.Quantum.QsCompiler
 open Microsoft.Quantum.QsCompiler.CompilationBuilder
 open Microsoft.Quantum.QsCompiler.DataTypes
 open Microsoft.Quantum.QsCompiler.SyntaxTree
-open Microsoft.Quantum.QsCompiler.Transformations.ClassicallyControlledTransformation
```

Comment:
Please revert the renaming for the Classically Controlled, Monomorphization, Monomorphization Validation, and Intrinsic Resolution transformation, as well as the changes to the Utility class so that the changes can be placed in a separate PR.